client/src/views/modals/select-records.js

```diff
@@ -255,6 +255,9 @@ define('views/modals/select-records', ['views/modal', 'search-manager'], functio
                 checkAllResultDisabled: !this.massRelateEnabled,
                 buttonsDisabled: true,
                 skipBuildRows: true,
+                pagination: this.getConfig().get('listPagination') ||
+                    this.getMetadata().get(['clientDefs', this.scope, 'listPagination']) ||
```

Comment:
I'd rather have it w/o clientDefs. So that clientDefs > listPagination was applicable only for the list view.

Reply:
I didn't pa attention to that 
sorry

Reply:
No problem. I merged.

Reply:
Thanks